From f3e8787dfc7fddb03ca62f7d7839d8c818fe97da Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Wed, 26 Oct 2016 18:25:09 +0300
Subject: [PATCH 1021/2241] arm64: defconfig: a80x0: a70x0: disable SATA
 address WA for A0

A7K/A8K rev A0 had special register mapping for SATA units,
so a Workaround for A0 was introduced to support that special mapping.

A1 revision doesn't use any special register mapping,
So this patch now disables the WA.

NOTE: with this change, SATA will not be supported for A0.

Change-Id: Ibe219997daf57cddd45e2da59b7917cb332381f5
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33378
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 42a130b..5feb658 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -95,7 +95,6 @@ CONFIG_SATA_AHCI=y
 CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_AHCI_CEVA=y
 CONFIG_AHCI_MVEBU=y
-CONFIG_CP110_SATA_ADDR_WA=y
 CONFIG_AHCI_XGENE=y
 CONFIG_PATA_PLATFORM=y
 CONFIG_PATA_OF_PLATFORM=y
-- 
2.7.4

